biomodels/src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -0,0 +1,65 @@
+package org.reactome.release;
+
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class ModelsTSVParserTester {
+    @Test
+    public void emptyMapFromEmptyFileName() {
+        Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("");
+        assertTrue("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void emptyMapFromUnknownFileName() {
+        Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("filedoesnotexist.tsv");
+        assertTrue("Unknown file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void emptyMapFromImproperFile() {
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/improper_models2pathways.tsv");
+        assertTrue("Improperly formatted file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void populatedMapFromPartiallyCorrectFile() {
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/partiallycorrect_models2pathways.tsv");
+        assertEquals(pathwayToBioModelsIds.keySet().size(), 3);
+        assertTrue(pathwayToBioModelsIds.get("R-MMU-1169091").contains("BIOMD0000000226"));
+    }
+
+    @Test
+    public void testModels2PathwaysCorrectlyParsed() {
```

Comment:
Could this test be divided up into smaller test since there are many test conditions?  It's usually best practice to have one assert statement per test whenever possible.  Common setup logic could be extracted to its own private method if needed.